libraries/Ticker/Ticker.h

```diff
@@ -96,10 +107,14 @@ class Ticker
 
 protected:	
 	void _attach_ms(uint32_t milliseconds, bool repeat, callback_with_arg_t callback, uint32_t arg);
+	static void internalCallback (void* arg);
 
 
 protected:
 	ETSTimer* _timer;
+	TickerFunction internalTicker = nullptr;
+	bool scheduleTicker = false;
```

Comment:
Instead of introducing a flag, can we do something like this for the "scheduled" case?
```c++
internalTicker = std::bind(schedule_function, tf);
```
and
```c++
internalTicker = tf;
```
for the normal (not scheduled)?